etc/FAUCET_VERSION

```diff
@@ -1 +1 @@
-1.9.48
+master
```

Comment:
I think the requisite change is in 1.9.50 that's currently available, so if that works we should just update here and make sure it all still works. I'm in the middle of testing 1.9.50 integration and it all seems good, but might as well get that tested with this too.

Reply:
changed to 1.9.50